doc/distrib/NodeHelpFiles/CoreNodeModels.FormattedStringFromArray.md

```diff
@@ -0,0 +1,17 @@
+## In Depth
+This node will convert an oject to a string. The second input `format specifier` controls how numeric inputs are converted to their string representations. 
+This `format specifier` inputs should be one of the c# standard format numeric specifiers.
+
+format specifiers should be in the form:
+`<specifier><precision>` for example F1
+
+Some commonly used format specifiers are:
+```
+G : general formatting G 1000.0 -> "1000"
+F : fixed point notation F4 1000.0 -> "1.0000"
+N : number N 1.0 -> 1000 -> "1,000.00"
```

Comment:
Did you mean this to be format specifier `N1`? If so, it should be 1000 -> 1,000.0

Reply:
so in an invariant culture `N` defaults to `N2` - but I should be clear since these are docs, I will update it.